gprofiler/main.py

```diff
@@ -127,6 +143,17 @@ def _generate_output_files(
                 .replace("{{{END_TIME}}}", end_ts)
             )
             Path(flamegraph_path).write_text(flamegraph_data)
+
+            # point last_flamegraph.html at the new file; and possibly, delete the previous one.
+            last_flamegraph = os.path.join(self._output_dir, "last_flamegraph.html")
+            prev_flamegraph = Path(last_flamegraph).resolve()
+            atomically_symlink(os.path.basename(flamegraph_path), last_flamegraph)
+            if self._rotating_output:
+                try:
+                    prev_flamegraph.unlink()
+                except FileNotFoundError:
+                    pass
+
```

Comment:
This code is duplicated, consider to create a function

Reply:
Yeah I thought about it but haven't done before I pushed. Will do